cloudinit/cmd/main.py

```diff
@@ -72,17 +72,21 @@ def print_exc(msg=""):
     sys.stderr.write("\n")
 
 
-def log_ppid():
-    if util.is_Linux():
+def log_ppid(distro):
+    if distro.is_linux:
         ppid = os.getppid()
-        LOG.info("PID [%s] started cloud-init.", ppid)
+        log = LOG.info
+        extra_message = ""
+        if 1 != ppid and distro.uses_systemd():
+            log = LOG.warning
+            extra_message = ("Not a supported configuration.",)
```

Comment:
Let's add details in the extra message to tell use what's unsupported about it.
`Unsupported configuration: boot stage called outside of systemd`